### DISCUSSION TITLE
Which technique were used to disassembly Zelda ROM?

### DISCUSSION THREAD

**Discussion by fernandoc1** [snesrev/zelda3#258] [Q&A]
Hi everyone!
I'm interested in knowing about which technique were used to disassembly zelda and mario roms to C code.
Is there any documentation?

**Comment by nicronex** [comment]
I think it's all done using Lakesnes.

**Reply by DPS2004** [reply]
Lakesnes is only used for emulating the PPU and APU as far as I know, and has nothing to do with how the disassembly was actually done.

**Comment by Nutzzz** [comment]
Check out https://github.com/IsoFrieze/DiztinGUIsh
